src/components/Panels/ElementsPanel/ElementsPanel.vue

```diff
@@ -0,0 +1,186 @@
+<template>
+    <div
```

Comment:
I can experience some lag when opening the panels when trying from the deploy preview, this doesn't seems to be the case on the production instance, can you check?

Reply:
Seemed like it was a common problem for many.
Solved it using this [reference](https://github.com/vuetifyjs/vuetify/issues/10140#issuecomment-605815875)